Widgets/widget-MyMentions.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "my-mentions",
+  "size": 10,
+  "fromVersion": "3.6.0",
+  "dataType": "messages",
+  "query": "type:mention*",
+  "name": "Mentions",
+  "dateRange": {
+    "period": { "byFrom": "days", "fromValue": null }
```

Comment:
same. our default is 7 days

Reply:
I want all time mentions (even if user hasn't logged in more than 7 days)